src/components/shared/SearchInput.js

```diff
@@ -141,6 +142,15 @@ class SearchInput extends Component<Props, State> {
     }
   };
 
+  renderSummaryMsg() {
+    const { summaryMsg, isLoading } = this.props;
+    if (summaryMsg) {
+      const loadingTextCss = isLoading ? " loading-indicator" : "";
+      return <div className={`summary${loadingTextCss}`}>{summaryMsg}</div>;
```

Comment:
could we come up with something that is descriptive of the styling we want, that is more general than loading. I think we might have other messages here in the future and want to re-use the styling

also you might want to checkout the classnames library